rasa/shared/core/training_data/story_reader/yaml_story_reader.py

```diff
@@ -362,7 +362,13 @@ def _parse_or_statement(self, step: Dict[Text, Any]) -> None:
     def _user_intent_from_step(
         self, step: Dict[Text, Any]
     ) -> Tuple[Text, Optional[Text]]:
-        user_intent = step.get(KEY_USER_INTENT, "").strip()
+        try:
+            user_intent = step.get(KEY_USER_INTENT, "").strip()
+        except AttributeError as e:
+            raise RasaException(
+                f"Missing intent value in {self._get_item_title()} step: {step} "
+                f"in '{self.source_name}'"
+            ) from e
```

Comment:
should we actually raise or treat this the same as the warning a few lines ahead? We don't raise for any other case so keeping it a warning seems more consistent to me

Reply:
True, I can change that - the question I have, when I use `raise_warning` would the next lines of code execute or do I need to add an empty return statement to stop the execution once this warning was raised?

Reply:
The next lines of code would execute. `raise_warning` should probably rather be something like `print_warning` to avoid that confusion